src/labs/labs-slice.ts

```diff
@@ -0,0 +1,42 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit'
```

Comment:
Need to add tests for this functionality. Of course the test implementation might change a little if we move it to the patient slice, however, the test concepts shouldn't. 

- [ ] test for the `fetchPatientLabs` thunk dispatching `fetchLabsStart`
- [ ] test that it calls the `LabRepository` correctly
- [ ] test that it dispatches `fetchLabsSuccesss` with the correct data
- [ ] test that the `fetchLabsStart` reducer update updates the is loading state correctly
- [ ] test that the `fetchLabsSuccess` reducer updates the labs list and loading state correctly